pkg/endpoint/bpf.go

```diff
@@ -468,23 +471,40 @@ func (e *Endpoint) regenerateBPF(owner Owner, epdir, reason string) error {
 		}
 	}
 
+	ctCleaned := make(chan struct{})
```

Comment:
Can you defer a `<-ctCleaned` here instead of making sure it's being added to all branches that exit?

Reply:
Done